components/bitcoin-topics/Navigation.tsx

```diff
@@ -40,7 +47,23 @@ export function Navigation({
     onLinkClick?: React.MouseEventHandler<HTMLAnchorElement>
 }) {
     const pathname = usePathname()
-    const posts = allCoreContent(sortPosts(allTopics))
+    const posts = useMemo(() => allCoreContent(sortPosts(allTopics)), [])
+    const [expandedTopics, setExpandedTopics] = useState<
+        Record<string, boolean>
+    >({})
+
+    // Load expanded state from localStorage on component mount
+    useEffect(() => {
+        const savedState = localStorage.getItem("expandedTopics")
```

Comment:
Why are we tracking and saving expanded sections?

I get the idea of resuming where the last stopped (good idea).
A couple of nits tho. 
- It doesn't track the exact topic they were on last.
- Multiple sections can be expanded as well, this doesn't provide much information.

Is there a specific need for the current implementation? If not we can remove it.

Reply:
Is there any reason we want to save the exact topic they were on last?

Having multiple expanded sections is beneficial ig, as it allows users to keep sections of interest open. This is also beneficial for users who are exploring or comparing different areas of content simultaneously

Reply:
I was thinking in line of user progress, so they can continue from where they last stopped.

Bitcoin topics layout has a hierarchy/progression structure and since we do not track user progression it would be nice to keep track of the last topic they were on. At least that was why I thought you had that in there.

Seems your implementation serves a different purpose.
And user progress is another conversation.

 > Having multiple expanded sections is beneficial ig, as it allows users to keep sections of interest open. This is also beneficial for users who are exploring or comparing different areas of content simultaneously
 
 It does this already right? in state. The extra bit of localStorage is what I struggle to see the benefits, that is over different browser sessions for the user. 
Extra features do not hurt so if you find it valuable then kindly keep it as is 🙏

Reply:
Ooooh, I understand your point now! That's a great approach, appreciate your suggestion.

Working on it and will push the update soon.

<img width="343" alt="image" src="https://github.com/user-attachments/assets/85915d0f-68c9-4ba7-a66b-773063491f68">